_data/feedhenry-repos.json

```diff
@@ -1,11 +1,4 @@
 {
-    "AeroGear-Push-Titanium": {
```

Comment:
This data will just get pulled back in when `_get_repo_data.py` is run again, so I'd remove this change. As long as you're not referencing "AeroGear-Push-Titanium" in any templates, it won't get shown.